introspection-engine/introspection-engine-tests/tests/tables/cockroachdb.rs

```diff
@@ -359,3 +326,28 @@ async fn a_table_with_non_id_autoincrement_cockroach(api: &TestApi) -> TestResul
 
     Ok(())
 }
+
+#[test_connector(tags(CockroachDb))]
+async fn introspecting_json_defaults_on_cockroach(api: &TestApi) -> TestResult {
+    let setup = indoc! {r#"
+       CREATE TABLE "A" (
+           id INTEGER NOT NULL Primary Key,
+           json Json Default '[]'::json,
```

Comment:
I guess it will always be on one line? (no multiline?)

Reply:
they get normalized when we get them back from the database